sigmf-spec.md

```diff
@@ -302,9 +302,9 @@ the Global object:
 | `version`      | true     | string  | The version of the SigMF specification used to create the Metadata file.|
 | `num_channels` | false    | uint    | Total number of interleaved channels in the dataset file. If omitted, this defaults to one.|
 | `sha512`       | false    | string  | The SHA512 hash of the Dataset file associated with the SigMF file.|
-| `offset`       | false    | uint    | The index number of the first sample in the Dataset. If not provided, this value defaults to zero. Typically used when a Recording is split over multiple files. All sample indices in SigMF are absolute, and so all other indices referenced in metadata for this recording should be greater than or equal to this value.|
+| `offset`       | false    | uint    | The index number of the first sample in the Dataset. If not provided, this value defaults to zero. Typically used when a Recording is split over multiple files. All sample indices in SigMF are absolute, and so all other indices referenced in metadata for this recording SHOULD be greater than or equal to this value.|
 | `description`  | false    | string  | A text description of the SigMF Recording.|
-| `author`       | false    | string  | The author's name (and optionally e-mail address) of the form "Bruce Wayne <wayne@example.com>".|
+| `author`       | false    | string  | The author's name (and OPTIONAL e-mail address) of the form "Bruce Wayne <wayne@example.com>".|
```

Comment:
maybe it should say something a bit more arbitrary like:
```Author's identifier potentially including name, handle, email, and/or other ID like Amateur Call Sign. For example "Bruce Wayne <bruce@waynetech.com>" or "Bruce (K3X)".```

Reply:
FYI I chose K3X since it's a special callsign that cannot actually exist. 3 letter call signs cannot end in X explicitly.

Reply:
I like this.